src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -78,6 +82,19 @@
     private final JobSchedulerService jobSchedulerService;
     private final CCDCreateCaseService ccdCreateCaseService;
 
+    private static BiFunction<ClaimSubmissionOperationIndicators, ClaimDocumentType, ClaimSubmissionOperationIndicators>
+        updateClaimSubmissionIndicatorByDocumentType = (indicator, docType) -> {
```

Comment:
Can we please add unit test for this?

Reply:
Suggestion: BiFunction usage here doesn't look a right choice here. A good candidate for its usage would be the assignment target for a lambda expression or method reference.